test/integration/generated/nameChecker/README.md

```diff
@@ -1,27 +1,87 @@
-## Azure SearchClient SDK for JavaScript
+# Azure SearchClient library for JavaScript
 
-This package contains an isomorphic SDK for SearchClient.
+This package contains an isomorphic SDK (runs both in node.js and in browsers) for Azure SearchClient.
+
+Client that can be used to query an index and upload, merge, or delete documents.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/undefined) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/search-documents) |
+[Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/undefined/samples)
```

Comment:
Are we taking the source code path as input parameter? The samples folder can be inferred from that.
And why don't we have the link to API ref docs here? That can be inferred from the package name right?

Reply:
Fixed.